api/app/users/api.py

```diff
@@ -545,3 +554,31 @@ def get(self):
             UserComment.user_id == args['user_id']).all()
 
         return comments
+
+class UserApplicationCommentReviewAPI(restful.Resource):
+    @auth_required
+    @marshal_with(user_comment_review_fields)
+    def get(self):
+        req_parser = reqparse.RequestParser()
+        req_parser.add_argument('event_id', type=int, required=True)
+        req_parser.add_argument('user_id', type=int, required=True)
+        args = req_parser.parse_args()
+
+        current_user = user_repository.get_by_id(g.current_user['id'])
+        if not current_user.is_event_admin(args['event_id']):
+            return FORBIDDEN
+
+        comments = db.session.query(UserComment).filter(
+            UserComment.event_id == args['event_id'],
+            UserComment.user_id == args['user_id']).all()
+        
+        review_by_user_firstname = user_repository.get_by_id(args['user_id']).firstname
+        
+        reviewer_id = db.session.query(ReviewResponse).filter(
+            ReviewResponse.reviewer_user_id == args['user_id'])
```

Comment:
I'm not sure what this code is doing?

Reply:
![image](https://user-images.githubusercontent.com/46738464/59963062-ed3f0100-94ed-11e9-9d0e-73dca3b9ffa0.png)

Reply:
Creates and  returns an object with 4 attributes.

Reply:
But we should not be  filtering by the reviewer_user_id = args['user_id']. 

A specific user creates a specific Response object (application response). (so you'll need to filter on user_id for Response)
ReviewResponse's are assigned to specific Response objects
Those are the reviews you need to pull
Reviewers are assigned to specific ReviewResponse's - reviewer_user_id - get the names of *those* users

And the call to get reviews for a specific user, should perhaps be in the reviews api.py, not under user?